ui/css/treeherder.css

```diff
@@ -94,164 +75,6 @@ input:focus::-moz-placeholder {
     width: 20px;
 }
 
-.navbar-right {
-    margin-right: 0px; /* override boostrap 3.3.5 behaviour -- doesn't work with the way we align our navbar */
-}
-
-#th-global-navbar-top {
-    padding-left: 0;
-    border-bottom: 1px solid black;
-}
-
-#th-logo {
-    padding-left: 14px;
-    border-left: 0;
-}
-
-th-watched-repo {
-    display: block;
-    float: left;
-}
-
-/* Override bootstrap for flush right navbar menus */
-.navbar-collapse {
-    padding-right: 0;
-}
-
-.nav-menu-btn {
-    margin-right: -4px;
-    padding-left: 14px;
-    padding-right: 14px;
-}
-
-.nav-help-btn {
-    margin-right: -4px;
-    padding-left: 12px;
-    padding-right: 8px;
-}
-
-.nav-help-icon {
-    font-size: 18px;
-    margin-right: 2px;
-}
-
-.nav-user-icon {
-    display: inline-block;
-    margin-right: 4px;
-    padding: 3px 4px;
-    vertical-align: text-bottom;
-    border-radius: 1px;
-    background: linear-gradient(#44adf3, #287cc2);
-}
-
-.nav-user-icon span:first-child {
-    margin: 0 auto;
-    font-size: 12px;
-    color: #fff;
-}
-
-.nav-login-btn {
-    padding-left: 22px;
-    padding-right: 27px;
-}
-
-.nav-persona-btn {
-    padding-left: 14px;
-    padding-right: 14px;
-}
-
-.watched-repo-main-btn {
-    border-right: 0;
-    padding-right: 5px;
-}
-
-.watched-repo-spacer {
-    padding-right: 7px;
-}
-
-.watched-repo-info-btn {
-    border-right: 0
-}
-
-.watched-repo-unwatch-btn {
-}
```

Comment:
Removed here since it had no properties. I left the class reference in the markup for automation purposes.